ReactApp/src/components/Editor.jsx

```diff
@@ -59,15 +59,15 @@ function Editor({ post }) {
 	const [registeredBlocks] = useState([]);
 	const [_isCodeEditorEnabled, setCodeEditorEnabled] = useState(false);
 	const titleRef = useRef();
-	const { addEntities, receiveEntityRecords } = useDispatch(coreStore);
 	const { setupEditor } = useDispatch(editorStore);
+	const { addEntities, receiveEntityRecords } = useDispatch(coreStore);
 
 	useEffect(() => {
 		window.editor = editor;
 		addEntities(postTypeEntities);
-		receiveEntityRecords('postType', post.type, post);
+		receiveEntityRecords('postType', post.type, post, undefined, true);
 
-		setupEditor(post, [], []);
+		setupEditor(post, {});
```

Comment:
This change was done in https://github.com/wordpress-mobile/GutenbergKit/pull/22 I'll update with `trunk` once it is merged.